test/typings/MappingProxy.test.ts

```diff
@@ -48,20 +59,6 @@ describe('Mapping proxy', () => {
         expect(pxthToString(defaultObserve.mock.calls[0][0])).toBe(pxthToString(createPxth(['b', 'b', 'd'])));
     });
 
-    it('observe/stopObserving (empty mapping path)', () => {
-        const proxy = new MappingProxy({ [RootPathToken]: createPxth(['a', 'd', 'c']) }, createPxth(['asdf']));
-
-        const defaultObserve = jest.fn();
-        const observer = jest.fn();
-
-        defaultObserve.mockReturnValue(0);
-
-        proxy.watch(createPxth(['asdf']), observer, defaultObserve);
-        expect(pxthToString(defaultObserve.mock.calls[0][0])).toBe(pxthToString(createPxth(['a', 'd', 'c'])));
-
-        defaultObserve.mockClear();
-    });
```

Comment:
Return this test back, as MappingProxySource allows to pass single path to whole object.